clemens-huebner/amos-ss16-proj8
# 60 add organizer

It is about adding registered user to organizer Collection to be organizer.
to test it : log in with Admin
go to Add Organizer in the menu 
put user email 
to check it go to baasbox / Collection organizer
note: this user story and u.s #49-Organizer-permission are complementary.
